arch/arm/core/swi_tables.ld

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#if LINKER_ZEPHYR_FINAL
+INCLUDE zephyr/isr_tables_swi.ld
```

Comment:
nothing wrong, as it appears to me that `isr_tables_swi.ld` is a clean ld snippet, meaning it doesn't need to be run through the pre-processor, but I would still like to know the rationale to use `INCLUDE` here (and let the linker do the inclusion) and then the pattern used elsewhere, which is `include()` and included by the preprocessor.
One advantage of doing it with pre-processor is that we end with a complete and self-contained final `linker.cmd` (and friends) file.

Reply:
I was trying to use include with preprocessor, the problem was that the file does not exist in a moment while the linker script is created. It will be created in build phase while it looks like the linker script is created during configuration phase of cmake.